server/logging_test.go

```diff
@@ -0,0 +1,184 @@
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+)
+
+type logOptionsTest map[string]string
+
+func TestEscapeDoubleQuotesOK(t *testing.T) {
+	t.Run("No quotes", func(t *testing.T) {
+		assert.Equal(t, "test", escapeDoubleQuotes("test"))
+	})
+	t.Run("One quote", func(t *testing.T) {
+		assert.Equal(t, "\\\"test", escapeDoubleQuotes("\"test"))
+	})
+	t.Run("Two quotes", func(t *testing.T) {
+		assert.Equal(t, "\\\"test\\\"", escapeDoubleQuotes("\"test\""))
+	})
+	t.Run("Three quotes", func(t *testing.T) {
+		assert.Equal(t, "\\\"\\\"test\\\"", escapeDoubleQuotes("\"\"test\""))
+	})
+	t.Run("Four quotes", func(t *testing.T) {
+		assert.Equal(t, "\\\"\\\"\\\"\\\"", escapeDoubleQuotes("\"\"\"\""))
+	})
+}
+
+func TestDefaultLoggingConsoleLogConfigOK(t *testing.T) {
+	var mapCfgEscaped = new(mlog.LoggerConfiguration)
+	var logOptionsConsole = new(logOptionsTest)
+
+	var targetCfg mlog.TargetCfg
+
+	t.Run("Unmarshall base correctly", func(t *testing.T) {
+		err := json.Unmarshal([]byte(defaultLoggingConsoleLogConfig()), mapCfgEscaped)
+		require.NoError(t, err)
+		require.NotNil(t, mapCfgEscaped)
+		targetCfg = (*mapCfgEscaped)["def"]
+		assert.Equal(t, "console", targetCfg.Type)
+		assert.Equal(t, "plain", targetCfg.Format)
+	})
+
+	t.Run("Unmarshall options correctly", func(t *testing.T) {
+		err := json.Unmarshal(targetCfg.Options, logOptionsConsole)
+		require.NoError(t, err)
+		require.NotNil(t, logOptionsConsole)
+		assert.Equal(t, "stdout", (*logOptionsConsole)["out"])
+	})
+}
+
+func TestDefaultLoggingFileLogConfigOK(t *testing.T) {
+	filename := randomString(10)
+	var mapCfgEscaped = new(mlog.LoggerConfiguration)
+	var logOptionsConsole = new(logOptionsTest)
+
+	var targetCfg mlog.TargetCfg
+
+	t.Run("Unmarshall base correctly", func(t *testing.T) {
+		err := json.Unmarshal([]byte(defaultLoggingFileLogConfig(filename)), mapCfgEscaped)
+		require.NoError(t, err)
+		require.NotNil(t, mapCfgEscaped)
+		targetCfg = (*mapCfgEscaped)["def"]
+		assert.Equal(t, "file", targetCfg.Type)
+		assert.Equal(t, "plain", targetCfg.Format)
+	})
+
+	t.Run("Unmarshall options correctly", func(t *testing.T) {
+		err := json.Unmarshal(targetCfg.Options, logOptionsConsole)
+		require.NoError(t, err)
+		require.NotNil(t, logOptionsConsole)
+		assert.Equal(t, filename, (*logOptionsConsole)["filename"])
+	})
+}
+
+func TestDefaultLoggingConfigOK(t *testing.T) {
+	t.Run("Console config is get correctly", func(t *testing.T) {
+		cfg := &ConfigPushProxy{
+			EnableFileLog: false,
+		}
+		assert.Equal(t, defaultLoggingConsoleLogConfig(), defaultLoggingConfig(cfg))
+	})
+
+	t.Run("File config is get correctly", func(t *testing.T) {
+		cfg := &ConfigPushProxy{
+			EnableFileLog:   true,
+			LogFileLocation: randomString(10),
+		}
+		assert.Equal(t, defaultLoggingFileLogConfig(cfg.LogFileLocation), defaultLoggingConfig(cfg))
+	})
+}
+
+func TestNewMlogLoggerConsoleLegacyOK(t *testing.T) {
```

Comment:
I see you still have all the `NewMlogLogger` related function in different tests. Can you join them under one test (with several runs as you have them now, of course).